src/Pages/Page.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Home from './Home/Home';
+import QuizPage from './Quiz/QuizPage';
+import Result from './Result/Results';
+
+export default function Page() {
+  return (
+    <div>
+      <Home />
```

Comment:
Why are you rendering these components at the same time? I guess it's not used, please remove it.

Reply:
Understood sir